hrm-domain/hrm-core/case/caseRoutesV0.ts

```diff
@@ -102,6 +112,44 @@ casesRouter.put('/:id/status', canEditCase, async (req, res) => {
   res.json(updatedCase);
 });
 
+casesRouter.put('/:id/status', canUpdateCaseStatus, async (req, res) => {
+  const {
+    accountSid,
+    user,
+    body: { status },
+  } = req;
+  const { id } = req.params;
+  const updatedCase = await caseApi.updateCaseStatus(id, status, accountSid, {
+    can: req.can,
+    user,
+  });
+  if (!updatedCase) {
+    throw createError(404);
+  }
+  res.json(updatedCase);
+});
```

Comment:
Is this endpoint duplicated?

Reply:
Looks like, not sure what happened there, nice catch!